fastapi_healthcheck/model.py

```diff
@@ -8,11 +8,11 @@
 class HealthCheckEntityModel(BaseModel):
     alias: str
     status: Union[HealthCheckStatusEnum, str] = HealthCheckStatusEnum.HEALTHY
-    timeTaken: Union[Optional[datetime], str]
+    timeTaken: Union[Optional[datetime], str] = ""
```

Comment:
Great fix! Too bad it is not merged :)

Reply:
try to use my fork + available/custom modules (like mine for cosmos/mongodb). maybe I do a new release with the fixes if I find some time

Reply:
@anesmemisevic Thanks for the PR.  Why are you looking to set a default value on TimeTaken to be a blank string?

Reply:
Because there were some errors from Pydantic but I don't remember at the moment... However, open to suggestions to change it to something more appropriate, or feel free to co-author.

Reply:
Done, thanks.